grapi/backend/kopano/message.py

```diff
@@ -164,11 +175,15 @@ def on_get_item(self, req, resp, folderid=None, itemid=None):
         Note:
             Based on MS Explorer result, it never validate folderid. So, we ignore it.
         """
+        store = req.context.server_store[1]
         if itemid is None:
             raise HTTPNotFound()
-        store = req.context.server_store[1]
-        item = _item(store, itemid)
-        self.respond(req, resp, item)
+        elif itemid == 'delta':  # TODO move to MailFolder resource somehow?
```

Comment:
for the `delta` endpoint, please move them in a separate method and then add their endpoints to the route list. So, it will be the `on_get_delta` method within this class and endpoints:
```
GET /me/mailFolders/{id}/messages/delta
GET /users/{id}/mailFolders/{id}/messages/delta
```
source: https://docs.microsoft.com/en-us/graph/api/message-delta?view=graph-rest-1.0&tabs=http#http-request

Reply:
Should be fixed in https://github.com/Kopano-dev/grapi/pull/4/commits/85a92a97405808ed00a36f4620d378b80b234acf